src/components/Header.js

```diff
@@ -29,8 +39,8 @@ const Header = () => {
                                 }
                                 />
                       <button className="search-btn" onClick={ () => {
-                        // const data= filterData(searchInput,);
-                         //setsearchInput(data);
+                        const data= filterData(searchInput,productList);
```

Comment:
data can be renamed to something meaningful.

Reply:
done